common-lib/src/main/scala/com/gu/mediaservice/model/UsageRights.scala

```diff
@@ -396,5 +397,12 @@ case class CreativeCommons(restrictions: Option[String] = None)
   }
 object CreativeCommons {
  implicit val jsonReads: Reads[CreativeCommons] = Json.reads[CreativeCommons]
- implicit val jsonWrites: Writes[CreativeCommons] = UsageRights.defaultWrites
+ implicit val jsonWrites: Writes[CreativeCommons] = (
```

Comment:
We can't use Json.writes[CreativeCommons] either as that then leaves out category et al as it only takes in the parameters of the case class.